packages/atomic/src/components/commerce/facets/atomic-commerce-facet/atomic-commerce-facet.new.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  wrapInCommerceInterface,
+  playExecuteFirstSearch,
+  playKeepOnlyFirstFacetOfType,
+} from '@coveo/atomic/storybookUtils/commerce-interface-wrapper';
+import {parameters} from '@coveo/atomic/storybookUtils/common-meta-parameters';
+import {renderComponent} from '@coveo/atomic/storybookUtils/render-component';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {html} from 'lit-html';
+
+const {play, decorator} = wrapInCommerceInterface({skipFirstSearch: true});
+
+const meta: Meta = {
+  component: 'atomic-commerce-facet',
+  title: 'Atomic-Commerce/Facet',
+  id: 'atomic-commerce-facet',
+  render: renderComponent,
+  decorators: [decorator],
+  parameters,
+  play,
+};
+
+export default meta;
+
+export const Default: Story = {
+  name: 'atomic-commerce-facet',
+  decorators: [
+    (_) => {
+      return html`<div id="code-root">
+        <atomic-commerce-facets></atomic-commerce-facets>
+      </div>`;
+    },
+  ],
+  play: async (context) => {
+    await play(context);
+    await playExecuteFirstSearch(context);
+    playKeepOnlyFirstFacetOfType('atomic-commerce-facet', context);
+  },
+};
```

Comment:
This component should never be used by users right ? The only reason to make a story for this is to test the individual component that gets automatically rendered and to show the  shadow parts. I think this is confusing for the documentation to have both `Facets/atomic-automatic-facets` and `Facet/atomic-automatic-facet`. I think we should both put them and in the same `Facets` folder and name this story "Shadow Parts" or something.

Reply:
We can tag some story "test only"

Reply:
@alexprudhomme btw atomic-automatic-facets is [supposed](https://docs.google.com/document/d/1oXGc9dJqBSQjlw7pLD6eolEuPvCyMUcAlyOhedbr78E/edit) to end up in `Interface Components/atomic-automatic-facets` later instead of in the Facets folder.

Reply:
All right, I understand we will review the documentation side later. This is focused on tests.